src/test/Makefile

```diff
@@ -58,9 +62,9 @@ baro_ms5611_unittest_DEFINES := \
                 USE_BARO_MS5611 \
                 USE_BARO_SPI_MS5611
 
-battery_unittest_SRC := \
-		$(USER_DIR)/sensors/battery.c \
-		$(USER_DIR)/common/maths.c
+#battery_unittest_SRC := \
```

Comment:
Same here.

Reply:
Same here, as well. [The test has a `.txt` suffix](https://github.com/betaflight/betaflight/blob/master/src/test/unit/battery_unittest.cc.txt) and therefore it did not execute anyway.

My change  was basically a no-op, just to satisfy the new consistency checks.

If you want, I can try to rename the test to `.cc` and check whether it would work. But when it's called `.cc.txt`, there must be a reason for disabling it that way.

Reply:
Investigated this one too. It misses `VBAT_SCALE_DEFAULT`, `VBAT_RESDIVVAL_DEFAULT` and a few other symbols. Some targets define those, some don't. They don't seem to be globally visible symbols (any more).

So that's why someone renamed it to `.txt`, I suppose.

Sorry, no clue what to do here. Both of tests had been broken + disabled before I even started looking into this.

Reply:
Ok, fair enough. In this case a comment stating that the test is disabled next to the commented out parts would have been helpful.

Reply:
So I've just added more comments to the disabled tests in #6760.